db/migrate/20211031132101_change_posts_constraints.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ChangePostsConstraints < ActiveRecord::Migration[6.0]
+  # rubocop:disable Metrics/MethodLength
+  def change
+    change_column_null(:posts, :title, false)
+    add_foreign_key :posts, :users, on_delete: :cascade
+
+    reversible do |dir|
+      dir.up do
+        execute <<-SQL.squish
+          ALTER TABLE posts
+            ADD CONSTRAINT check_posts_title_length
+              CHECK (char_length(title) BETWEEN 5 AND 255);
```

Comment:
@sudakov-eugene I'm not quite sure about keeping it 5 anymore, I would say that 1 is enough.

Reply:
This value looks quite adequate to me, otherwise what title can you create with one char? @rakvium